src/plugin/places/abstractkmlmanager.js

```diff
@@ -0,0 +1,472 @@
+goog.module('plugin.places.AbstractKMLManager');
```

Comment:
This class should live under the `plugin.file.kml` namespace since it no longer has anything Places-specific in it. Within the plugin hierarchy, we prefer to keep classes in the more generic namespaces if possible.